src/test/suite/commands/new-line.test.ts

```diff
@@ -119,13 +119,7 @@ suite("newLine", () => {
           assert.strictEqual(activeTextEditor.selection.active.character, 4);
         });
 
-        const languagesAutoDoc = [
-          // "c", "cpp"  // Auto-indent for doc comments does not work with these languages in test env while I don't know why...
```

Comment:
Auto indentation for C/C++ is provided by the [Better Comments extension](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments).